flask/app/routes.py

```diff
@@ -83,9 +83,9 @@ def login():
 @routes.route("/api/logout", methods=["POST"])
 @login_required
 def logout():
-    if not request.json:
-        app.logger.error("Logout failed as request body is not JSON")
-        abort(400, description="Request body must be JSON!")
+    if request.headers.get("Content-Type") != "application/json":
+        app.logger.error("Logout failed as Content-Type is not JSON")
+        abort(400, description="Content-Type must be JSON!")
```

Comment:
I think we can update this status code to 415 to be consistent with the rest.

Reply:
updated in [1412c73](https://github.com/ccmbioinfo/stager/pull/484/commits/1412c73c9bf5de331aa6ccac700c43f117706816#diff-5ccfa28a10586b56ab898a8aac81bd2a24552a0de717b1a1bc982205f6be0c37R88).